models/org.go

```diff
@@ -62,13 +62,9 @@ func (org *User) getTeams(e Engine) error {
 		Find(&org.Teams)
 }
 
-// GetTeams returns paginated teams that belong to organization.
-func (org *User) GetTeams(opts *SearchTeamOptions) error {
-	if opts.Page != 0 {
-		return org.getTeams(opts.getPaginatedSession())
-	}
-
-	return org.getTeams(x)
+// LoadTeams load teams if not loaded.
+func (org *User) LoadTeams() error {
+	return org.loadTeams(x)
```

Comment:
Assume user has joined over 1000 teams? Or we should limit user max joined teams number.

Reply:
this org.getTeams is exactly used as other struct.**loadIssues/loadX/loadY/..** functions so I renamed it.
removing it because of speed improvements is not in scope of this pull ... but If you wish I'll add a todo with an open issue to this line?

Reply:
Yes, I would like to leave it to another PR and not remove the paniation.